src/middlewared/middlewared/plugins/crypto_/cert_info.py

```diff
@@ -47,17 +47,6 @@ async def ec_curve_choices(self):
         """
         return {k: k for k in EC_CURVES}
 
-    @accepts()
```

Comment:
Are we certain UI is not using these ?

Reply:
Yes, this is hardcoded. We can call an endpoint if we feel like that would be beneficial.